doc/usecases/cli/UC_get_config.md

```diff
@@ -0,0 +1,22 @@
+# Use Case: cli get configuration
+
+## Summary
+
+- **Scope:** CLI use
+- **Level:** Developer Goal, UI
```

Comment:
Please use one of the following values, as defined in the template https://github.com/ElektraInitiative/libelektra/blob/master/doc/usecases/template.md:

- Cloud
- Kite
- Sea
- Underwater
- Clam

See this PR for details: https://github.com/ElektraInitiative/libelektra/pull/4762/files#r1045125432

This applies also to the two other use cases.